_posts/blog/2016-27-06-holacrcy-part-1.md

```diff
@@ -0,0 +1,96 @@
+
+ * A purpose 
+ * Domains to control
+ * Set of accountabilities and corresponding domains
+ Only the purpose is mandatory, the accountabilities and domains can evolve over time. As an example you can have a look at the developer role from GlassFrog - 
+ which is a web based tool developed by HolacracyOne. 
+ 
+**Developer**
+**Purpose**
+Satisfy the customer, with Product Vision as proxy, through early and continuous delivery of truly valuable software
+
+**Accountabilities**
+
+ * Implementing GlassFrog user stories, bug fixes, and chores in alignment with: 
+ * development practices set by Scrum Master 
+ * implementation standards set by Quality Guardian
+ * Testing new functionality before each production deployment
+ * Providing input into estimation at the request of Scrum Master
+ * Sharing development challenges with Product Vision, API Product Designer, and Developer as challenges arise
+ * Demonstrating working software to Product Vision regularly and frequently, and integrating feedback surfaced by Product Vision
+ 
+A definition like this makes it explicit what you can expect from a developer and on the other hand a dev 
+knows what she/he is accountable for. 
+But as the title of this sections says, there is a big difference between the persons who are working in 
+an organization and the roles they fill. Decoupling these two elements allows improving the culture of 
+an organization drastically. Conflicts in an organization are based on clashes on roles that are 
+involved but they are mistaken them for clashes of the people who are filling the roles. This makes the 
+conflicts personal and we try to resolve them by smoothing the relationships and miss the opportunity 
+to resolve the underlying issue in the organization. For example, if I as an developer need time to 
+unit test a feature this may clash with a product owner who wants a feature out as soon as possible. So 
+it seems that she/he is not trusting your judgement. But in fact the conflict isn’t personnel at all, 
+both of your are just filling your role and there is a tension that can be used to improve the organization 
+even further. HolacrcyOne provides a nice video that explains nicely what a tension can be:
+
+### What is a tension
+<div class="yt-video"><iframe width="560" height="315" src="https://www.youtube.com/embed/MUHfVoQUj54" frameborder="0" allowfullscreen></iframe></div>
+
+How does now the circles fit into the picture? Well actually a circle is just an accumulation of some 
+roles that are all working together on a broader purpose. So basically you can say that if a role 
+becomes too much for one person - it becomes a circle. It also follows the same rules - needs a purpose 
+and a domain that it controls. In other words a circle has the autonomy and authority to self-organize 
+all roles inside the circle. <br><br>   
+This self-organization follows strict rules - which are called governance, which I will explain in my 
```

Comment:
...rules - called governance. (emphasize governance)  But those will be explained in my next post.